Gems/Multiplayer/Code/Source/Debug/MultiplayerDebugSystemComponent.cpp

```diff
@@ -133,8 +134,13 @@ namespace Multiplayer
                 if (auto console = AZ::Interface<AZ::IConsole>::Get())
                 {
                     const MultiplayerAgentType multiplayerAgentType = multiplayerInterface->GetAgentType();
-                    const bool enableHosting = multiplayerAgentType == MultiplayerAgentType::Uninitialized;                    
-                    if (ImGui::BeginMenu(HostLevelMenuTitle, enableHosting))
+
+                    // Enable the host level selection menu if we're neither a host nor client, or if we are hosting, but haven't loaded a level yet.
+                    const bool isLevelLoaded = AzFramework::LevelSystemLifecycleInterface::Get()->IsLevelLoaded();
+                    const bool isHosting = multiplayerAgentType == MultiplayerAgentType::ClientServer || multiplayerAgentType == MultiplayerAgentType::DedicatedServer;
```

Comment:
nit: Consider adding parenthesis to make line easier for maintainers to read

Reply:
fixed! thank you :)